google/cloud/aiplatform/helpers/container_uri_builders.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import re
+
+from google.cloud.aiplatform.constants import prediction
+from google.cloud.aiplatform.initializer import global_config
+
+
+def get_prediction_container_uri(
+    framework: str,
+    framework_version: str,
+    region: str = None,
+    with_accelerator: bool = False,
```

Comment:
I'm unsure about using a flag here, since [with PyTorch containers](https://cloud.google.com/vertex-ai/docs/training/pre-built-containers#pytorch) - there are three flavors `xla`, `cpu`, and `gpu`. But as this seems version dependent, the method could handle searching for the correct URI.

Reply:
I think we should accept `cpu`, `gpu` for now to make this more extendable if we support other accelerators in the future.